ls1intum/Artemis
# Filter solution information in quizzes after the first submission was done

### Checklist

### Motivation and Context


Student should not get information about the correct quiz before the exam is over

### Description

Filter informations from StudentExam before the exam is over - after the last student submitted -> the information is included again, so that it can be correctly displayed when the student sees the review

### Steps for Testing


1. Participate in an exam with MC, DnD and SA questions
2. After the first save of the questions, reload the exam
3. Check the server response for information about the solution -> you shouldn't find any information

### Screenshots
